experimenter/experimenter/experiments/tests/factories.py

```diff
@@ -468,6 +468,9 @@ class NimbusExperimentFactory(factory.django.DjangoModelFactory):
     qa_status = factory.LazyAttribute(
         lambda o: random.choice(list(NimbusExperiment.QAStatus)).value
     )
+    is_firefox_labs_opt_in = factory.LazyAttribute(lambda o: False)
+    firefox_labs_title = factory.LazyAttribute(lambda o: faker.text(200))
+    firefox_labs_description = factory.LazyAttribute(lambda o: faker.text(200))
```

Comment:
200 characters is huge for what goes in these fields. lets do `slugify(faker.catchphrase()`